### ISSUE TITLE
Change default units from "b,Kb,Mb" to " B, KB, MB"

### ISSUE THREAD

**Issue by dalboris** [Ionaru/easy-markdown-editor#239]
First of all, thanks for the great library, it is very useful!

Not sure if this issue truly qualifies as a "bug", but almost. The default value for `sizeUnits` is currently `b,Kb,Mb`. I find this extremely confusing: in English-speaking countries these mean "bits" instead of "bytes", and thus the displayed values are wrong by an 8-fold multiplying factor. The correct English abbreviations for bytes, kilobytes, and megabytes are supposed to be B, kB, and MB, see for example:

https://en.wikipedia.org/wiki/Kilobyte

In fact, the values displayed by easyMDE seem to actually be kibibytes and mebibytes (KiB, MiB), but this is probably too confusing for most users. I'd advocate for B, KB, and MB as default (note the capital K, unlike the official abbreviation), which is how the English versions of Windows and macOS display them.

Finally, note that most operating systems display a space between the value and the unit (e.g., `198.7 KB`), but easyMDE doesn't use such space (e.g., `198.7KB`). It would be nice to have that space, so I believe the most appropriate default value should be ` B, KB, MB` (note the three whitespace characters).

I have already overridden this in my configuration (see below), and it's really nice that this is customizable. It'd just be even better if this was the default value, I think.

```
var easyMDE = new EasyMDE({
  ...
  imageTexts: {
      ...
      sizeUnits: ' B, KB, MB'
  }
});
```

Thanks!

**Comment by Ionaru** [comment]
Sounds good to me. Feel free to submit a PR with these changes, or I'll do it at the end of the week. :)

**Comment by Ionaru** [comment]
Closed by #242
